nj-derive/src/ast/util.rs

```diff
@@ -28,6 +32,21 @@ impl TypePathUtil for TypePath {
             .find(|_| true)
             .map(|segment| &segment.ident)
     }
+    /// find lifetime name.
+    fn lifetime(&self) -> Option<&Lifetime> {
+        let first = self.path.segments.first()?;
+        let lifetime_arg = if let PathArguments::AngleBracketed(arguments) = &first.arguments {
+            arguments.args.first()?
```

Comment:
Does this only handle cases where there is a single lifetime?

Reply:
Ah. Good catch.